cmake/std/trilinosprhelpers/unittests/trilinos_pr_test.ini

```diff
@@ -222,6 +223,18 @@ setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
 
 
 
+[Trilinos_pullrequest_intel_19.0.5]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 6.1.0
+module-load sems-intel: 19.0.5
+module-load sems-mpich: 3.2
+use SEMS-DEFAULT:
+use ATDM-DEFAULT:
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
```

Comment:
Same question as above. If we are using 6.1.0, why wouldn't we use that for the local version also?

Reply:
If I recall, this is supposed to change what the sems modules expect, but does not. I would have to experiment.

Reply:
Ya, I've not seen it make a difference when putting together a build.
But I never dug further into it.
I can set them to the associated used GCC version and we can worry about the details in another issue.

Reply:
If we set it to the associated GCC, I wonder if we need it at all. It would seem the purpose of such an option might be to specify a compiler version different than the version of the compiler being used. For example, there are times when you have a set of TPLs built with a different version of a compiler that is link-compatible with the compiler you are using, but is not the same compiler. Could we just take this out?

Reply:
That's sort of what I was wondering.
I can remove them all and see how it tests out.